EIPS/eip-2159.md

```diff
@@ -0,0 +1,56 @@
+| ethereum_best_known_block_number | Gauge       | The estimated highest block available                             | `highestBlock` of `eth_syncing` or `eth_blockNumber` if not syncing | 
+| ethereum_peer_count              | Gauge       | The current number of peers connected                             | `net_peerCount`                                                     |
+| ethereum_peer_limit              | Gauge       | The maximum number of peers this node allows to connect           | No equivalent                                                       |
+
+Note that `ethereum_best_known_block_number` always has a value. When the `eth_syncing` JSON-RPC method would return `false`, the current chain height is used.
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+The defined metrics are independent of Ethereum client implementation but provide sufficient information to create an overview dashboard to support monitoring a group of Ethereum nodes.
+
+There is a similar, though more prescriptive, specification for [beacon chain client metrics](https://github.com/ethereum/eth2.0-metrics/blob/master/metrics.md).
+The specific details of how to expose the metrics has been omitted as there is variance in existing implementations and standardising this does not provide any significant benefit.
+
+## Backwards Compatibility
+<!--All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
+This is *not* a consensus affecting change.
+
+Clients may already be publishing these metrics using different names and changing to the new form may break existing alerts or dashboards. Clients that want to avoid this incompatibility can expose the metrics under both the old and new names.
+
+
+## Implementation
+<!--The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.-->
+These metrics are currently captured by [Pantheon](http://pegays.tech) under `pantheon_` prefixed names. They will be renamed to match this EIP in the future.
```

Comment:
The link seems to be not working.

Reply:
Well that's just embarrassing.  Sorry, fixed now.